octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/EntityFieldsComposite.java

```diff
@@ -135,9 +138,9 @@ private void drawEntityFields(EntityModelWrapper entityModelWrapper) {
 
     private void drawEntityFields(Set<String> shownFields, EntityModelWrapper entityModelWrapper) {
         Arrays.stream(fieldsComposite.getChildren())
-                .filter(child -> child != null)
-                .filter(child -> !child.isDisposed())
-                .forEach(child -> child.dispose());
+        .filter(child -> child != null)
+        .filter(child -> !child.isDisposed())
+        .forEach(child -> child.dispose());
```

Comment:
@atothhpe I'm not familiar with this part, but is disposing of the child element also make sure to clear the fieldsComposite structure? Shouldn't we also clear the null and isDisposed children from the data structure?

Reply:
@paul-ciobanu in swt getChildren() returns all the child controls of a composite control. If you dispose a control, it will be removed from the UI. Actually the issue here is that the check for null and !disposed is paranoid. Since get children can't return null or disposed widgets